### PR TITLE
UCT/IB: Improve async event error message.

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://bgate.mellanox.com/jenkins/job/gh-ucx-pr/1027/ for details.

**Comment by brminich** [comment]
:+1:

File: src/uct/ib/base/ib_device.c [hunk 1]
```diff
@@ -128,15 +128,21 @@ static void uct_ib_async_event_handler(int fd, void *arg)
...
+                 "DCT key violation", event.element.dct->dct_num);
+        break;
     case IBV_EXP_EVENT_DCT_ACCESS_ERR:
+        snprintf(event_info, sizeof(event_info), "%s on DCTN 0x%x",
+                 "DCT access error", event.element.dct->dct_num);
+        break;
     case IBV_EXP_EVENT_DCT_REQ_ERR:
         snprintf(event_info, sizeof(event_info), "%s on DCTN 0x%x",
-                 ibv_event_type_str(event.event_type), event.element.dct->dct_num);
+                 "DCT requester error", event.element.dct->dct_num);
```
Comment (shamisp) [inline src/uct/ib/base/ib_device.c]: I would think that ibv_event_type_str() info is still useful.

Comment (yosefe) [inline src/uct/ib/base/ib_device.c]: it doesn't work for _EXP

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/2989/ for details (Mellanox internal link).

**Pull request merged**